kubernetes_platform/go.mod

```diff
@@ -1,15 +1,12 @@
 module github.com/kubeflow/pipelines/kubernetes_platform
 
-go 1.16
+go 1.21
 
-require (
```

Comment:
@mprahl could you please confirm that you ran a `go mod tidy` for the go.mod changes here? Just want to make sure since I don't see a corresponding `go.sum` file.

Reply:
Good catch! I forgot to commit it.